scaling_performance/optimizing_storage.adoc

```diff
@@ -126,5 +126,22 @@ Overlay is also supported for Docker use cases as of Red Hat Enterprise Linux
 7.2, and provides faster start up time and page cache sharing, which can
 potentially improve density by reducing overall memory utilization.
 
-
-
+[[benefits-of-using-the-overlay-graph-driver]]
+== Benefits of Using the Overlay Graph Driver with SELinux
+
+The default Docker storage configuration on Red Hat Enterprise Linux (RHEL)
+continues to be device-mapper. As of RHEL 7.2, Overlay became a supported graph
+driver. As of RHEL 7.4, SELinux and the Overlay2 graph driver became a supported
+combination.
+
+The main advantage of the Overlay file system is Linux page cache sharing among
+containers on the same node. This attribute of Overlay leads to reduced
+input/output (I/O) during container startup (and, thus, faster container startup
+time by several hundred milliseconds), as well as reduced memory usage when
+similar images are running on a node. Both side-effects lend themselves well to
```

Comment:
"Both side-effects lend themselves well" seems too marketing-speak to me. Not sure what to replace it with. "These effects are beneficial in many environments..." ??